roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
```

Comment:
on Atomic Host there are currently two versions of docker: docker-current and docker-latest (which I think only incidentally are the same on Atomic Host 7.3.2) and I'd say they can still be different in future:

https://access.redhat.com/articles/2317361

I'd suggest to get directly the version from /usr/bin/docker (that you can see it is a shell script that calls the "real" docker executable) as:

docker version --format '{{json .Server}}'

and get the version from there